symfony/webpack-encore-bundle/2.0/webpack.config.js

```diff
@@ -37,7 +37,8 @@ Encore
      * https://symfony.com/doc/current/frontend.html#adding-more-features
      */
     .cleanupOutputBeforeBuild()
-    .enableBuildNotifications()
+    // Displays build status system notifications to the user
+    // .enableBuildNotifications()
```

Comment:
Just two newlines for visual separations with the rest, and that's good to me :)

```suggestion

    // Displays build status system notifications to the user
    // .enableBuildNotifications()

```